packages/contracts/contracts/test/CommonTest.t.sol

```diff
@@ -145,26 +145,50 @@ contract L2OutputOracle_Initializer is CommonTest {
         oracle = L2OutputOracle(address(new Proxy(multisig)));
         vm.label(address(oracle), "L2OutputOracle");
 
+        ResourceMetering.ResourceConfig memory config = Constants.DEFAULT_RESOURCE_CONFIG();
+        systemConfig = new SystemConfig({
+            _owner: address(1),
+            _overhead: 0,
+            _scalar: 10000,
+            _batcherHash: bytes32(0),
+            _gasLimit: 30_000_000,
+            _unsafeBlockSigner: address(0),
+            _config: config
+        });
+
         // Deploy the ValidatorPool
         poolImpl = new ValidatorPool({
             _l2OutputOracle: oracle,
             _trustedValidator: trusted,
             _minBondAmount: minBond
         });
 
+        // Mock KromaPortal
+        KromaPortal mockPortal = new KromaPortal({
+            _l2Oracle: oracle,
+            _guardian: guardian,
+            _paused: false,
+            _config: systemConfig,
+            _zkMerkleTrie: ZKMerkleTrie(address(0))
+        });
```

Comment:
Do we still use `PortalInitializer`?

Reply:
For the KromaPortal test, a ZKMerkleTrie contract is required. But the other tests don't, so it need to separate them.

Reply:
As L2OO requries KromaPortal, things get complicated. Let's tackle the issue later.